Missionframework/scripts/shared/functions/F_getGroupType.sqf

```diff
@@ -20,44 +20,41 @@ if ((_grouptype == 'infantry') && (_vehicletype != '')) then {
 	} foreach heavy_vehicles;
 
 	if ( _grouptype == 'infantry' ) then {
-	{
-		if  ( _vehicletype == (_x select 0)) then {
-			if ( _vehicletype in uavs ) then {
-				_grouptype = 'uav';
-			} else {
+		{
+			if  ( _vehicletype == (_x select 0)) then {
 				_grouptype = 'air';
 			};
-		};
-	} foreach air_vehicles;
+		} foreach air_vehicles;
 	};
 
 	if ( _grouptype == 'infantry' ) then {
-	{
-		if  ( _vehicletype == (_x select 0)) then {
-			if ( _vehicletype in uavs ) then {
-				_grouptype = 'uav';
-			} else {
+		{
+			if  ( _vehicletype == (_x select 0)) then {
 				_grouptype = 'light';
 			};
-		};
-	} foreach light_vehicles;
+		} foreach light_vehicles;
 	};
 
 
 	if ( _grouptype == 'infantry' ) then {
-	{
-		if  ( _vehicletype == (_x select 0)) then {
-			_grouptype = 'support';
-		};
-	} foreach support_vehicles;
+		{
+			if  ( _vehicletype == (_x select 0)) then {
+				_grouptype = 'support';
+			};
+		} foreach support_vehicles;
 	};
 
 	if ( _grouptype == 'infantry' ) then {
-	{
-		if  ( _vehicletype == (_x select 0)) then {
-			_grouptype = 'static';
-		};
-	} foreach static_vehicles;
+		{
+			if  ( _vehicletype == (_x select 0)) then {
+				_grouptype = 'static';
+			};
+		} foreach static_vehicles;
+	};
+
+	// Check if vehicle config says it's an UAV, if it is always set its _grouptype to 'uav'
+	if ( (_vehicletype call F_isClassUAV) ) then {
```

Comment:
I can't say why they've started with that. Maybe you do it by a reason or do you just orientate on the "other format"?
I'm speaking of the additional spaces after brackets

    if ( ( some bool ) )

instead of

    if ((some bool))

And you don't need the second bracket pair here :)

Reply:
To be honest I just wanted to conform to format used in the file.

You are right these brackets are not needed here. Sometimes when writing SQF im a bit too defensive with stuff like that because I'm not very experienced with this language😞  and it's better to have working code with few more brackets than not working code.

Feel free to reformat this. Maintainer commits are enabled.

Reply:
I'm also using one bracket pair more in ifs with multiple conditions etc. but in this "one condition" if, you don't need to worry :)